src/plugins/intel_gna/frontend/layer_quantizer.hpp

```diff
@@ -4,99 +4,91 @@
 
 #pragma once
 
-#include <string>
-#include <utility>
-#include <cmath>
-
-#include "backend/gna_types.h"
-#include "gna_plugin_log.hpp"
-#include "quantized_layer_params.hpp"
-#include "quantization.h"
-#include "gna_graph_tools.hpp"
-#include "blob_factory.hpp"
-#include "precision_ex.hpp"
+#include "ie_precision.hpp"
 #include "layers/gna_layer_info.hpp"
+#include "quantized_layer_params.hpp"
+#include "backend/gna_types.h"
+#include "quantization.hpp"
 #include "weights_converter.hpp"
-#include <legacy/layer_transform.hpp>
+
 
 namespace GNAPluginNS {
 namespace frontend {
 
-/**
- * @brief description of quantisation precision
- * @tparam Ip - input precision
- * @tparam Op - output precision
- * @tparam Wp - weights precision
- * @tparam Bp - biases precision
- * @tparam Np - network precision - can be auto generated in future
- */
-template <class Ip, class Op, class Wp, class Bp, class Np>
-struct QuantDescTmpl {
-    using WeightsPrecision = Wp;
-    using BiasesPrecision = Bp;
-
-    InferenceEngine::TPrecision<Ip> _Ip;
-    InferenceEngine::TPrecision<Op> _Op;
-    InferenceEngine::TPrecision<Wp> _Wp;
-    InferenceEngine::TPrecision<Bp> _Bp;
-
-    QuantDescTmpl() = default;
-    QuantDescTmpl(InferenceEngine::TPrecision<Ip> _Ip,
-              InferenceEngine::TPrecision<Op> _Op,
-              InferenceEngine::TPrecision<Wp> _Wp,
-              InferenceEngine::TPrecision<Bp> _Bp) : _Op(_Op), _Ip(_Ip), _Wp(_Wp), _Bp(_Bp) {
-    }
+// TODO: POT AAQ support can be added by modifying these functions
+static InferenceEngine::Precision GetInputPrecision() {
+    return InferenceEngine::Precision::I16;
```

Comment:
This is intentional for now, because, as you noted, int8 input precision was a dead code. Most probably I'll extend this function in a following PR, based on test results.

Reply:
@sirzabek That what I said @dorloff when this code was going to be merged. But for some reasons it was done, and that's why I'm asking.

Reply:
Most of the merged code will stay there as it had been originally merged. With this PR the quantizer pair for low precision was removed and temporarily not replaced by anything else, but low precision input might be enabled finally if we can prove the accuracy with POT path is acceptable. So I seemed for me sensible to neither remove the whole low precision code nor enable it fully at the moment yet.